projects/media-viewer/src/lib/media-viewer/toolbar/search-bar/search-bar.component.ts

```diff
@@ -27,7 +27,7 @@ export class SearchBarComponent {
     if (e.code === 'F3' || (e.ctrlKey && e.code === 'KeyF')) {
       e.preventDefault();
 
-      this.searchBarHidden.next(false);
+      this.searchBarHidden = false;
```

Comment:
Are you are about this? It won't inform any other listeners to the subject?

Reply:
restored